src/next-child-view-container.js

```diff
@@ -0,0 +1,129 @@
+import _ from 'underscore';
+import emulateCollection from './utils/emulate-collection';
+
+// Provide a container to store, retrieve and
+// shut down child views.
+const Container = function() {
+  this._views = [];
+  this._viewsByCid = {};
+  this._indexByModel = {};
+  this._updateLength();
+};
+
+emulateCollection(Container.prototype, '_views');
+
+function stringComparator(comparator, view) {
+  return view.model && view.model.get(comparator);
+}
+
+// Container Methods
+// -----------------
+
+_.extend(Container.prototype, {
+
+  // Add a view to this container. Stores the view
+  // by `cid` and makes it searchable by the model
+  // cid (and model itself). Additionally it stores
+  // the view by index in the _views array
+  _add(view, index) {
```

Comment:
```js
_add(view, index = this._views.length) {
```
and delete line https://github.com/marionettejs/backbone.marionette/pull/3244/files#diff-24babd314506fb676ab7fdb23980804cR40